tests/test_compiler.py

```diff
@@ -13,9 +14,9 @@
     if ".cache" not in str(p) and not p.is_dir() and p.suffix == ".sol"
 ]
 TEST_CONTRACTS = [str(p.stem) for p in TEST_CONTRACT_PATHS]
+PATTERN_REQUIRING_COMMIT_HASH = re.compile(r"\d+.\d+.\d+\+commit.[\d|a-f]+")
```

Comment:
The `.` should be escaped so they match `"."` explicitly rather than any character